python/nano/src/bigdl/nano/pytorch/strategies/ddp_spawn.py

```diff
@@ -75,8 +74,8 @@ def __init__(self, strategy: 'DDPSpawnStrategy') -> None:   # type: ignore[overr
     def launch(self, function: Callable, *args: Any,
                trainer: Optional["pl.Trainer"] = None, **kwargs: Any) -> Any:
         # pytorch_lightning 1.6 uses this method to create child processes
-        invalidInputError(trainer is not None and self._strategy.cluster_environment is not None,
-                          'strategy.cluster_environment and trainer cannot be None')
+        invalidInputError(self._strategy.cluster_environment is not None,
+                          'strategy.cluster_environment cannot be None')
```

Comment:
Maybe adding in comments in which case the `strategy.cluster_environment` will be None? 

And also telling what the users should do if he/she meets this error in the error message (if he should not see this error, maybe we can say "you should raise an issue in BigDL.").

Reply:
in fact, the cluster_environment cannot be None in any situations